posix/clone

```diff
@@ -103,3 +103,10 @@ tag)
 	clone-commit
 	;;
 esac
+
+for path in ${DRONE_OUTPUT_FILE_PATHS_CONTENT//,/ }
+do
+    # call your procedure/other scripts here below
+    FILE_CONTENT=`cat $path`
+    echo "$path=$FILE_CONTENT" >> $DRONE_OUTPUT
```

Comment:
Multi-line outputs don't work well currently. What's the usecase here?

Reply:
we want to resolve expressions in manager, so we want to send some files to manager using plugin outputs.

Reply:
@shubham149 Changed the approach to use base64 as discussed. I didn't encode key. Encoded only content.